do files/01_Generate survey data/1.clean_dynata_and_lucid.do

```diff
@@ -42,12 +42,20 @@ Otras situaciones
 (Other situations)
 */
 
+* Replicator addition - to preserve original variable before recoding
+gen militaryoriginal = militaryservice
 
+* Replicator's note - All respondents who included 1 as an option are coded as 1
 replace militaryservice="1" if substr(militaryservice,1,1)=="1"
```

Comment:
This line replaces `military service` with simply "1" for every observation that starts with "1", even if other options are selected; it looks like this leads to the liberal coding because this edit comes before any of the other options are checked.